src/main/java/com/factly/dega/web/rest/OrganizationResource.java

```diff
@@ -97,16 +101,22 @@ public OrganizationResource(OrganizationService organizationService) {
     /**
      * GET  /organizations : get all the organizations.
      *
+     * @param keycloakId the keycloakId of the user
      * @param pageable the pagination information
      * @return the ResponseEntity with status 200 (OK) and the list of organizations in body
      */
     @GetMapping("/organizations")
     @Timed
-    public ResponseEntity<List<OrganizationDTO>> getAllOrganizations(Pageable pageable, HttpServletRequest request) {
-        log.debug("REST request to get a page of Organizations");
-        Page<OrganizationDTO> page = organizationService.findAll(pageable);
-        HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/organizations");
-        return ResponseEntity.ok().headers(headers).body(page.getContent());
+    public ResponseEntity<List<OrganizationDTO>> getOrganizations(@RequestParam(value = "keycloakId", required = false) String keycloakId,
```

Comment:
rename keycloakId to keycloakUserId as it can be confused for organization's keycloakId

Reply:
done